### PR TITLE
[RAPTOR-7419] Split drum java predictor into 3 jars

### PR DESCRIPTION
**Pull request by yakov-g**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Split java predictor into 3 jars.
- drum-base-predictor.jar will be uploaded to maven (sonatype repo we use is experiencing  super high load)
- drum-py4j-entrypoint.jar and drum-predictors.jar will use  drum-base-predictor.jar  as dependency.

### CODE REVIEW

**Comment by yakov-g** [comment]
jarvis test cmrunner_integration

**Comment by yakov-g** [comment]
jarvis run cmrunner_integration

**Comment by yakov-g** [comment]
jarvis run cmrunner_integration

**Comment by yakov-g** [comment]
jarvis run cmrunner_integration

**Comment by yakov-g** [comment]
jarvis test cmrunner_integration

**Comment by yakov-g** [comment]
jarvis test cmrunner_integration

**Comment by yakov-g** [comment]
jarvis test cmrunner_integration

**Comment by yakov-g** [comment]
jarvis test functional_drop_in

**Comment by yakov-g** [comment]
jarvis test functional_drop_in

**Comment by yakov-g** [comment]
jarvis test functional_drop_in

**Comment by yakov-g** [comment]
jarvis test functional_drop_in

**Comment by yakov-g** [comment]
jarvis test functional_drop_in

File: tests/drum/custom_java_predictor/pom.xml [hunk 1]
```diff
@@ -26,9 +26,9 @@
             <version>2.7</version>
         </dependency>
         <dependency>
-            <groupId>com.datarobot.drum</groupId>
-            <artifactId>predictors</artifactId>
-            <version>1.0</version>
+            <groupId>com.datarobot</groupId>
+            <artifactId>drum-base-predictor</artifactId>
+            <version>1.1.0</version>
```
Comment (alexey-kry) [inline tests/drum/custom_java_predictor/pom.xml]: hm... `rum-base-predictor` version is `1.0.0` in `custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/base_predictor/pom.xml` and as a dependency in all other pom files. 
How did this build?

**Review by alexey-kry — CHANGES_REQUESTED** [review]

File: tests/drum/custom_java_predictor/pom.xml (hunk 1 above)
Comment (yakov-g) [reply tests/drum/custom_java_predictor/pom.xml]: I noticed and update this :)

**Comment by yakov-g** [comment]
jarvis test cmrunner_integration

**Review by timsetsfire — APPROVED** [review]

**Review by alexey-kry — APPROVED** [review]

**Pull request merged**